frontend/src/metabase/nav/containers/MainNavbar/SidebarItems/SidebarOnboardingSection/SidebarOnboardingSection.tsx

```diff
@@ -165,28 +160,14 @@ export function SidebarOnboardingSection({
       bottom={0}
       pos="sticky"
       bg="bg-white"
-      className={cx({ [CS.borderTop]: collapseCTASection })}
+      className={cx({ [CS.borderTop]: showCTASection })}
```

Comment:
With and without the CTA
![image](https://github.com/user-attachments/assets/b634f899-9be9-447b-9a67-70a62e677afc)

Reply:
With and without the CTA (scrolled)
![image](https://github.com/user-attachments/assets/abf14c8c-36a1-436a-ad57-cbdd3f5355ac)